modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-new-3x-major.adoc

```diff
@@ -184,6 +184,27 @@ In version 3.x of this connector, the polling mode of the Subscriber source is s
 </anypoint-mq:subscriber>
 ----
 
+This table lists the 2.x `polling` parameters and their 3.x equivalents.
+
+
+[%header,cols="20a,30a,20a,30a"]
+|===
+| Version 2.x Parameter |Description| Version 3.x Equivalent |Description
+|`fetchSize`| Declared at the `config` level, controls the size of the local message buffer, which is calculated as three times the `fetchSize` value.
+|`fetchSize`| Declared at the `Subscriber` level, sets the maximum number (1-10) of messages to fetch on each polling execution.
+The default is 10.
+
```

Comment:
Just checking: both parameters named the same, right?

Reply:
Yes.